kci_build

```diff
@@ -289,21 +291,23 @@ or
             return False
         return kernelci.build.install_kernel(
             args.kdir, tree_name, tree_url, branch, args.commit, args.describe,
-            args.describe_verbose, args.output, args.publish_path)
+            args.describe_verbose, args.output, args.publish_path,
+            args.install_path, args.mod_path)
 
 
 class cmd_push_kernel(Command):
     help = "Push the kernel binaries"
-    args = [Args.kdir, Args.api, Args.token]
+    args = [Args.kdir, Args.api, Args.token, Args.install_path]
```

Comment:
Why not do the same as with other commands and use it as an optional argument?

```python
opt_args = [Args.install_path]
```

Reply:
Thanks for pointing this out.
I'm still mostly hacking my way through ... I did not get that difference before

Reply:
And sorry for noise, I'm pushing things to my branch for concourse to pick it up ... I forgot this it updates the PR each time

Reply:
No problem :)